insights/specs/__init__.py

```diff
@@ -471,7 +471,7 @@ class Specs(SpecSet):
     root_crontab = RegistryPoint()
     route = RegistryPoint()
     rpm_V_packages = RegistryPoint()
-    rsyslog_conf = RegistryPoint(filterable=True)
+    rsyslog_conf = RegistryPoint()
```

Comment:
Since this spec was already filtered, this change will have to go through the approval process.